skimage/transform_xy/_warps.py

```diff
@@ -208,7 +208,7 @@ def rescale(image, scale, order=1, mode='reflect', cval=0, clip=True,
     ----------------
     order : int, optional
         The order of the spline interpolation, default is 1. The order has to
-        be in the range 0-5. See `skimage.transform.warp` for detail.
+        be in the range 0-5. See `skimage.transform_xy.warp` for detail.
```

Comment:
I'm concerned about the documentation. Generally, when we deprecate, we want to point to the *new* way of doing things. The only reason to keep the old way hanging around is for backwards compatibility. But anyone new who checks the docs should be pointed to the One True Way of doing things, and that's the new way.